src/sections/Blog/Blog-sidebar/blogSidebar.style.js

```diff
@@ -183,6 +183,7 @@ const BlogSideBarWrapper = styled.div`
     .explain-1 {
         padding-top: 0rem;
         text-align: center;
+        margin-top: -2rem;
```

Comment:
Instead of adding a `margin-top` here, remove the top padding from the callout used in the sidebar at line: 190

Reply:
@Nikhil-Ladha Thanks for the suggestion 👍I have changed, kindly check it again